deployment/conf/.templates/deployment.cfg.templ

```diff
@@ -0,0 +1,6 @@
+[authserv2]
```

Comment:
This section will need to be renamed to match the section the service is looking for, and the other directives from the deploy.cfg copied in

Reply:
Looks like this is the example deploy.cfg, just change authserv2 to assemblyhomology. Looks like you changed the temp dir to an assemblyhomology related name

[assemblyhomology]

# MongoDB information.
mongo-host=
mongo-db=
mongo-user=
mongo-pwd=

# A directory to use for temporary files:
temp-dir=./assemblyhomology_temp

Reply:
No, any temp directory you want to use is fine. And ignore my comments about the other configs - the config file looked a little sparse, and but I hadn't seen that the deploy.cfg.example was just as sparse